doc/source/user_guide/skimage2_migration.md

```diff
@@ -8,24 +8,92 @@ This document is a work in progress and still subject to change.
 
 scikit-image is preparing to release version 2.0 as a new package: `skimage2`.
 Alongside skimage2, we will release version 1.0.0. Versions 1.x will be using the current API.
-Versions 1.1.x will throw a FutureWarning upon import, as a means to notify users that
+Versions 1.1.x will throw a `FutureWarning` upon import, as a means to notify users that
 they should either upgrade to skimage2 or pin to version 1.0.x.
 
 We have undertaken this to make some long-outstanding, backward-incomptible changes to the scikit-image API.
 Most changes were difficult or impossible to make using deprecations alone.
-To honor the Hinsen principle—that is, never change results silently, unless to fix a bug—we introduce a new package, which gives users an explicit way of upgrading.
+To honor the Hinsen principle (that is, never change results silently unless to fix a bug), we introduce a new package, which gives users an explicit way of upgrading.
 Users also have the option to use the two versions side-by-side while they do so.
 
 You can find a more detailed description of our motivation and discussion leading up to this in {doc}`SKIP 4 <../skips/4-transition-to-v2>`.
 
-## Changes in skimage2
+(enable-skimage2-warnings)=
 
-_To be defined._
+## Enable skimage2-related warnings
+
+Even before skimage2 is released, you may enable skimage2-related warnings to prepare for code changes early on.
+Run the following [warnings filter](https://docs.python.org/3/library/warnings.html#the-warnings-filter) before you use scikit-image in your code:
+
+```python
+import warnings
+import skimage as ski
+warnings.filterwarnings(action="default", category=ski.util.PendingSkimage2Change)
+```
+
+This will raise a warning for any piece of code which needs updating to continue working with the new (skimage2) API the exact same way it used to (in the old API).
+
+## Updating existing code
+
+When switching to the new namespace in version 2.0, some code needs to be updated to keep working as before.
+
+:::{note}
+Because these changes live in a new namespace, your code will _not_ break automatically if you don't explicitly change your imports and start importing from `skimage2`!
+:::
+
+(threshold-blob-funcs)=
+
+### Range-preserving thresholding in `feature.blob_*` functions
+
+**Affects:**
+
+- {func}`skimage.feature.blob_dog`
+- {func}`skimage.feature.blob_doh`
+- {func}`skimage.feature.blob_log`
+
+**Description:**
+
+Starting with **version 0.26**, the value range of the parameter `image` is always preserved.
```

Comment:
```suggestion
Starting with **version 0.26**, the value range of the input `image` is always preserved.
```
At this point, `image` is an argument, and the backticks around 'image' make it clear that it's the name of a parameter for the above-listed functions.